src/main/java/org/mule/tools/apikit/misc/ExampleUtils.java

```diff
@@ -47,8 +47,14 @@ public static String getExampleAsJSONIfNeeded(String payload) {
   public static String getDataWeaveExpressionText(String example) {
     String transformContentType = getExampleContentType(example);
     example = getExampleAsJSONIfNeeded(example);
+    String weaveResult;
 
-    final String weaveResult = asDataWeave(example, transformContentType);
+    try {
+      weaveResult = asDataWeave(example, transformContentType);
+    } catch (Exception e) { // Fallback to text/plain content type
+      transformContentType = TEXT_PLAIN_CONTENT_TYPE;
```

Comment:
cant we do this in the asDataWeave method? what exception is this throwing?

Reply:
You're right, I've just commited what I think is cleaner approach